client/src/main/java/com/kritsit/casetracker/client/domain/services/Editor.java

```diff
@@ -109,7 +109,7 @@ public Staff getUser() {
 
     public List<Person> getComplainants() {
         List<Person> complainants = new ArrayList<>();
-        for (Case c : cases) {
+        for (Case c : getCases()) {
```

Comment:
Get cases is called straight after initialisation, but this is definitely a cleaner solution that does not rely on the programmer getting it right